packages/wxt/vitest.globalSetup.ts

```diff
@@ -0,0 +1,27 @@
+import { rm } from 'fs/promises';
+import { existsSync } from 'fs';
+import consola from 'consola';
+
+let teardownHappened = false;
+
+export async function setup() {
+  // @ts-expect-error
+  globalThis.__ENTRYPOINT__ = 'test';
+}
+
+export async function teardown() {
+  if (teardownHappened) {
+    throw new Error('teardown called twice');
+  }
+  teardownHappened = true;
+
+  const e2eDistPath = './e2e/dist/';
+  if (existsSync(e2eDistPath)) {
+    try {
+      await rm(e2eDistPath, { recursive: true, force: true });
+      consola.info(`cleaned up ${e2eDistPath}`);
+    } catch (error) {
+      consola.error(error);
+    }
+  }
+}
```

Comment:
The globalSetup and teardown was prepared as a side note. `e2e/dist` had swelled up to nearly GB.

Reply:
Let's extract this change to a separate PR, but it's a good fix. I thought I was doing this already though :/ maybe I'm thinking of a different project lol

Reply:
Haha common things. Okay, I'll separate this and submit!